app/assets/javascripts/reducers/course.js

```diff
@@ -29,10 +29,15 @@ const initialState = {
   end: null,
   timeline_start: null,
   timeline_end: null,
+  language: 'en',
+  project: 'wikipedia',
+  home_wiki: { language: 'en', project: 'wikipedia' },
```

Comment:
You can define a `defaultWiki` const above here, and then use it in both home_wiki and `wikis`.